main.js

```diff
@@ -199,6 +199,7 @@
 	 */
 	(function () {
 		var $window = $(window);
+		var mq = window.matchMedia( "(min-width: 672px)" );
```

Comment:
Maybe check for window.matchMedia:

    window.matchMedia && window.matchMedia('(min-width: 672px)');